Copying.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>I love it, but what was the revenge for? :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
for killing my rainbow fish with her shark! thats totally her fault :D